test/clientside/run.js

```diff
@@ -5,27 +5,51 @@ var browserify =  require('browserify');
 var proxyquire =  require('../..');
 var vm         =  require('vm');
 var test       =  require('tape');
+var ES6Promise =  require('es6-promise').Promise;
```

Comment:
This was just used to make sure that all the tests were compiled before attempt to run tape, to avoid the race conditions